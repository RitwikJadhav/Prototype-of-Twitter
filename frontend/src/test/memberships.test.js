import React from 'react';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Memberships from '.././components/lists-module/memberships';
import renderer from 'react-test-renderer';

Enzyme.configure({ adpater: new Adapter() });

it('should render Membership Page correctly', () => {
    const component = renderer.create(<Memberships />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});