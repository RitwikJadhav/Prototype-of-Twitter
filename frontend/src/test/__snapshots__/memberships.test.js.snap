// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Membership Page correctly 1`] = `
<div>
  <h4>
    Membered lists
  </h4>
</div>
`;
